library/src/containers/Schemas/SchemaExample.tsx

```diff
@@ -1,42 +1,51 @@
-import React, { Component } from 'react';
+import React from 'react';
 
+import { CodeComponent, Badge, BadgeType } from '../../components';
+
+import { bemClasses, generateExample } from '../../helpers';
 import { Schema } from '../../types';
-import { generateExample } from '../../helpers/generateExampleSchema';
-import { CodeComponent, GeneratedBadge } from '../../components';
-import { SchemaExample } from './styled';
-import { GENERATED_BADGE } from '../../constants';
+import { SCHEMA_EXAMPLE_TEXT } from '../../constants';
+
 interface Props {
   title?: string;
   schema: Schema;
 }
 
-export class SchemaExampleComponent extends Component<Props> {
-  render() {
-    const { title, schema } = this.props;
-    const example = JSON.stringify(
-      schema.example ? schema.example : generateExample(schema),
-      null,
-      2,
-    );
-
-    if (!example) {
-      return null;
-    }
+export const SchemaExampleComponent: React.FunctionComponent<Props> = ({
+  title,
+  schema,
+}) => {
+  const example = JSON.stringify(
+    schema.example ? schema.example : generateExample(schema),
+    null,
+    2,
+  );
 
-    return (
-      <SchemaExample>
-        <CodeComponent
-          code={example}
-          title={
-            <>
-              {title ? title : 'Example'}{' '}
-              {!schema.example && (
-                <GeneratedBadge>{GENERATED_BADGE}</GeneratedBadge>
-              )}
-            </>
-          }
-        />
-      </SchemaExample>
-    );
+  if (!example) {
+    return null;
   }
-}
+
+  return (
+    <div className={bemClasses.element(`schema-example`)}>
```

Comment:
Yes, but please see above comment.

Reply:
here I see something different than in comment above

Reply:
I mean about wrapping generic component with additional `div` as in above for easier styling. I know that it could done with passing className as prop but this solution is more readable (for me of course).